i18n/en.json

```diff
@@ -376,5 +376,39 @@
   "cs-scrabble-count": "high-scoring Scrabble® words",
   "cs-bucket-emptyslot": "[empty]",
   "homepage-cs-link": "View bot information",
-  "zuzakistan-full": "The Federated Microstates of Zuzakistan"
+  "zuzakistan-full": "The Federated Microstates of Zuzakistan",
+  "/referenda": "/referenda",
```

Comment:
These should be documented in `i18n/qqx.json`.
Incidentally, you can use e.g. `zuzakistan.com?uselang=qqx` to force viewing the i18n keys.